scripts/check_unit.sh

```diff
@@ -31,5 +36,11 @@ PKGS=`go list github.com/hyperledger/aries-framework-go/cmd/aries-agent-rest/...
                                                  grep -v /mocks`
 go test $PKGS -count=1 -race -coverprofile=profile.out -covermode=atomic -timeout=10m
 amend_coverage_file
-
 cd "$pwd" || exit
+
+
+# Running wasm unit test
+# TODO Support collecting code coverage  https://github.com/agnivade/wasmbrowsertest/issues/5
+PKGS="github.com/hyperledger/aries-framework-go/pkg/storage/jsindexeddb"
+GOOS=js GOARCH=wasm go test $PKGS -count=1 -exec=wasmbrowsertest -cover  -timeout=10m
```

Comment:
I assume you are creating a separate JS test runner (as this will require chrome to be installed)?

Reply:
done